deployments/core.yml

```
@@ -145,6 +149,9 @@ Mappings:
 
 Conditions:
   AttachLayers: !Not [!Equals [!Join ['', !Ref LayerVersionArns], '']]
+  KvProvisioningEnabled: !Equals
+    - !Ref KvTableBillingMode
+    - PROVISIONED
```

Nit: I think it's fine to keep this in one line:

```suggestion
  KvProvisioningEnabled: !Equals [!Ref KvTableBillingMode, PROVISIONED]
```